apps/api/package.json

```diff
@@ -50,7 +50,8 @@
     "reflect-metadata": "^0.2.0",
     "rxjs": "^7.8.1",
     "socket.io": "^4.7.5",
-    "uuid": "^9.0.1"
+    "uuid": "^9.0.1",
+    "typescript": "^5.1.3"
```

Comment:
Why would we need typescript in here? That too not in dev dependency?

Reply:
we need ts for ts 👀
I have put it here because for docker I don't wanna install devDeps, as they are not required for prod